swagger.json

```diff
@@ -3007,6 +3087,64 @@
             }
           }
         },
+        "FleetVehicleResponse": {
+            "type": "object",
+            "description": "A vehicle object as returned for fleet/vehicle",
+            "required": [
+                "id",
+                "name"
+            ],
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the vehicle.",
+                    "format": "int64",
+                    "example": 112
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Name of the vehicle.",
+                    "example": "Truck A7"
+                },
+                "vehicleInfo": {
```

Comment:
Nit: Does this endpoint already exist and return `vehicleInfo`? This seems like an unnecessary envelope to me.

Reply:
Yeah, the endpoint returns a list of devices like 
```graphql
type Device struct {
	Id          int64
	Name        string
	VehicleInfo struct {
		Vin   string
		Make  string
		Model string
		Year  int64
	}
	HarshAccelSetting int64
	ExternalIds       []struct {
		Key   string
		Value string
	}
}
```